packages/core/src/Modules/Cashier/Containers/account-transfer.jsx

```diff
@@ -31,19 +32,31 @@ const AccountTransfer = ({
     setIsTransferConfirm,
     setSideNotes,
 }) => {
+    const [is_loading_status, setIsLoadingStatus] = React.useState(true);
     React.useEffect(() => {
         setActiveTab(container);
         onMount();
+
+        BinarySocket.wait('authorize', 'website_status', 'get_settings', 'paymentagent_list').then(() => {
```

Comment:
Can we use `WS` from 'Services/ws-methods' instead? It refers to the same thing, but I noticed we mostly use WS in react components and we use BinarySocker directly in .js files only.

Reply:
Yes sure.